src/RoomsSDKAdapter.js

```diff
@@ -0,0 +1,62 @@
+import {Observable} from 'rxjs';
+import {RoomsAdapter} from '@webex/components';
+
+export const ROOM_UPDATED_EVENT = 'updated';
+
+/**
+ * The `RoomsSDKAdapter` is an implementation of the `RoomsAdapter` interface.
+ * This adapter utilizes the Webex JS SDK to fetch data about a room.
+ *
+ * @export
+ * @class RoomsSDKAdapter
+ * @extends {RoomsAdapter}
+ */
+export default class RoomsSDKAdapter extends RoomsAdapter {
+  /**
+   * Fetches the room data from the sdk and returns in the shape required by adapter.
+   *
+   * @param {string} ID
+   * @returns {Room}
+   * @memberof RoomsSDKAdapter
+   */
+  async fetchRoom(ID) {
+    const room = await this.datasource.rooms.get(ID);
```

Comment:
```js
const {id, title, type} = await this.datasource.rooms.get(ID);
```

Reply:
I can fix this, but these kind of review comments should not be PR blockers.